decidim-core/config/locales/en.yml

```diff
@@ -1283,6 +1266,34 @@ en:
         close_window: Close window
         share: Share
         share_link: Share link
+    statistics:
+      answers_count: Answerss
+      assemblies_count: Assemblies
+      comments_count: Comments
+      conferences_count: Conferences
+      consultations_count: Consultations
+      debates_count: Debates
+      elections_count: Elections
+      endorsements_count: Endorsements
+      followers_count: Followers
+      headline: Statistics
+      meetings_count: Meetings
+      no_stats: There are no statistics yet.
+      orders_count: Supports
+      pages_count: Pages
+      participants_count: Participants
+      posts_count: Posts
+      processes_count: Processes
+      projects_count: Projects
+      proposals_accepted: Accepted Proposals
+      proposals_count: Proposals
+      results_count: Results
+      sortitions_count: Sortitions
+      supports_count: Supports
+      surveys_count: Surveys
```

Comment:
I'm not sure about all this block. Ideally, `decidim-core` shouldn't need to know how to name the `proposals_accepted` statistics, for example, so each module should be able to dofine the `decidim.statistics.foobar_count` key in their own i18n file.

Was there any technical issue in doing it this way? If not, could you move the keys to each module, please? 🙏

Reply:
Hi @mrcasals, No there's not a technical reason. Yes I agree that each module should have its translations, so I'll leave here only the ones related to `decidim-core` 😁